Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics/MicroGraphics/Samples/ImageLoadSample/ImageLoadSample.csproj

```diff
@@ -29,9 +29,11 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\..\..\..\..\Meadow.Core\source\Meadow.F7\Meadow.F7.csproj" />
```

Comment:
This one should also probably stay as local refs

Reply:
problem is with CI - it doesn't have that project, so it fails to build.